config/openshift/role.yaml

```diff
@@ -1,259 +1,49 @@
-  - namespaces
-  verbs:
-  - get
-  - list
-  - create
-  - update
-  - delete
-  - patch
-  - watch
-- apiGroups:
-  - apps
-  resources:
-  - deployments
-  - daemonsets
-  - replicasets
-  - statefulsets
-  - deployments/finalizers
-  verbs:
-  - '*'
-- apiGroups:
-  - monitoring.coreos.com
-  resources:
-  - servicemonitors
-  verbs:
-  - get
-  - create
-  - delete
-- apiGroups:
-  - rbac.authorization.k8s.io
-  resources:
-  - clusterroles
-  - roles
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
-  - serviceaccounts
-  verbs:
-  - get
-  - list
-  - create
-  - update
-  - delete
-  - patch
-  - watch
-- apiGroups:
-  - rbac.authorization.k8s.io
-  resources:
-  - clusterrolebindings
-  - rolebindings
-  verbs:
-  - get
-  - create
-  - update
-  - delete
-- apiGroups:
-  - apiextensions.k8s.io
-  resources:
-  - customresourcedefinitions
-  - customresourcedefinitions/status
-  verbs:
-  - get
-  - create
-  - update
-  - delete
-  - list
-  - patch
-  - watch
-- apiGroups:
-  - admissionregistration.k8s.io
-  resources:
-  - mutatingwebhookconfigurations
-  - validatingwebhookconfigurations
-  verbs:
-  - get
-  - list
-  - create
-  - update
-  - delete
-  - patch
-  - watch
-- apiGroups:
-  - build.knative.dev
-  resources:
-  - builds
-  - buildtemplates
-  - clusterbuildtemplates
-  verbs:
-  - get
-  - list
-  - create
-  - update
-  - delete
-  - patch
-  - watch
-- apiGroups:
-  - extensions
-  resources:
-  - deployments
-  verbs:
-  - get
-  - list
-  - create
-  - update
-  - delete
-  - patch
-  - watch
-- apiGroups:
-  - extensions
-  resources:
-  - deployments/finalizers
-  verbs:
-  - get
-  - list
-  - create
-  - update
-  - delete
-  - patch
-  - watch
-- apiGroups:
-  - policy
-  resources:
-  - podsecuritypolicies
-  verbs:
-  - get
-  - create
-  - update
-  - delete
-  - use
-- apiGroups:
-  - operator.tekton.dev
-  resources:
-  - '*'
-  - tektonaddons
-  verbs:
-  - '*'
-- apiGroups:
-  - tekton.dev
-  - triggers.tekton.dev
-  - operator.tekton.dev
-  resources:
-  - '*'
-  verbs:
-  - '*'
-- apiGroups:
-  - dashboard.tekton.dev
-  resources:
-  - '*'
-  - tektonaddons
-  verbs:
-  - '*'
-- apiGroups:
-  - security.openshift.io
-  resources:
-  - securitycontextconstraints
-  verbs:
-  - use
-- apiGroups:
-  - route.openshift.io
-  resources:
-  - routes
-  verbs:
-  - get
-  - list
-- apiGroups:
-  - coordination.k8s.io
-  resources:
-  - leases
-  verbs:
-  - get
-  - list
-  - create
-  - update
-  - delete
-  - patch
-  - watch
-- apiGroups:
-  - console.openshift.io
-  resources:
-  - consoleyamlsamples
-  - consoleclidownloads
-  - consolequickstarts
-  verbs:
-  - '*'
-- apiGroups:
-  - autoscaling
-  resources:
-  - horizontalpodautoscalers
-  verbs:
-  - '*'
-- apiGroups:
-  - policy
-  resources:
-  - poddisruptionbudgets
-  verbs:
-  - '*'
-- apiGroups:
-  - monitoring.coreos.com
-  resources:
-  - servicemonitors
-  verbs:
-  - '*'
-- apiGroups:
-  - batch
-  resources:
-  - jobs
```

Comment:
If i remember correctly we need this apiGroups for openshift cleanup jobs

Reply:
true.
i haven't removed anything, just simplified it.
you can see the job permissions on line 20: https://github.com/tektoncd/operator/blob/08bb35ee591f0e7f7faccc9c9e967e38a9cb9b9f/config/openshift/role.yaml#L20

Reply:
yes, i did change the order :)

Reply:
I did a diff, looks correct to me.

Reply:
Ah i see got it :+1: 

I missed that :grin: